tcpproxy.go

```diff
@@ -91,6 +97,24 @@ func (p *TCPProxy) CloseConns() {
 	}
 }
 
+// PauseConns stops traffic flowing through all the connections that are
```

Comment:
Actually, it pauses all traffic, so perhaps:

// PauseConns stops all traffic flowing through the proxy.

?